addons/project-folders/addon.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "Project Folders",
+  "description": "Put your projects in a folder to access them easily.",
+  "credits": [
+    {
+      "name": "Platformer444",
+      "link": "https://scratch.mit.edu/users/Platformer444"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/mystuff"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["https://scratch.mit.edu/mystuff"]
+    }
+  ],
+  "info": [
```

Comment:
Could a performance notice be added since the addon still takes a while to make all the requests?